package.json

```diff
@@ -1,65 +1,99 @@
 {
-  "private": true,
-  "name": "jsxtreme-markdown",
-  "license": "MIT",
-  "scripts": {
-    "start-repl": "webpack-dev-server --config repl/webpack.config.js --content-base repl",
-    "build-repl": "PRODUCTION=true webpack --config repl/webpack.config.js",
-    "lint": "eslint .",
-    "test-jest": "jest",
-    "format": "prettier --write '**/*.js'",
-    "postinstall": "lerna bootstrap",
-    "precommit": "lint-staged",
-    "pretest": "npm run lint",
-    "test": "jest"
-  },
-  "devDependencies": {
-    "@mapbox/assembly": "^0.21.2",
-    "babel-loader": "^7.1.4",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-react": "^6.24.1",
-    "eslint": "^4.19.1",
-    "eslint-plugin-node": "^6.0.1",
-    "husky": "^0.14.3",
-    "jest": "^23.1.0",
-    "lerna": "^2.9.0",
-    "lint-staged": "^7.0.4",
-    "prettier": "1.13.7",
-    "webpack": "^3.4.1",
-    "webpack-dev-server": "^2.6.1",
-    "webpack-parallel-uglify-plugin": "^1.1.0"
-  },
-  "engines": {
-    "node": ">=4"
-  },
-  "jest": {
-    "coverageReporters": [
-      "text",
-      "html"
-    ],
-    "roots": [
-      "<rootDir>/packages"
-    ],
-    "testMatch": [
-      "**/*.test.js"
-    ],
-    "clearMocks": true,
-    "testEnvironment": "node"
-  },
-  "prettier": {
-    "singleQuote": true
-  },
-  "lint-staged": {
-    "*.js": [
-      "eslint",
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "babel": {
-    "presets": [
-      "env",
-      "react"
-    ]
-  }
+	"private": true,
+	"name": "jsxtreme-markdown",
+	"license": "MIT",
+	"scripts": {
+		"start-repl": "webpack-dev-server --config repl/webpack.config.js --content-base repl",
+		"build-repl": "PRODUCTION=true webpack --config repl/webpack.config.js",
+		"lint": "eslint .",
+		"test-jest": "jest",
+		"format": "prettier --write '**/*.js'",
+		"postinstall": "lerna bootstrap",
+		"precommit": "lint-staged",
+		"pretest": "npm run lint",
+		"test": "jest"
+	},
+	"engines": {
+		"node": ">=10"
+	},
+	"jest": {
+		"coverageReporters": [
+			"text",
+			"html"
+		],
+		"roots": [
+			"<rootDir>/packages"
+		],
+		"testMatch": [
+			"**/*.test.js"
+		],
+		"clearMocks": true,
+		"testEnvironment": "node"
+	},
+	"prettier": {
+		"singleQuote": true
+	},
+	"lint-staged": {
+		"*.js": [
+			"eslint",
+			"prettier --write"
+		]
+	},
+	"dependencies": {
+		"@babel/code-frame": "^7.10.1",
+		"@babel/core": "^7.10.2",
+		"@babel/parser": "^7.10.2",
+		"@mapbox/hast-util-table-cell-style": "^0.1.3",
+		"balanced-match": "^1.0.0",
+		"block-elements": "^1.2.0",
+		"del": "^5.1.0",
+		"front-matter": "^4.0.2",
+		"github-slugger": "^1.3.0",
+		"hastscript": "^5.1.2",
+		"js-beautify": "^1.11.0",
+		"kebab-case": "^1.0.0",
+		"line-column": "^1.0.2",
+		"loader-utils": "^2.0.0",
+		"lodash": "^4.17.15",
+		"lodash.clonedeep": "^4.5.0",
```

Comment:
Can `lodash.clonedeep` be removed here or does the main package need to have it's all dependencies installed (due to the monorepo structure)? I see that it's used and installed in jsxtereme-markdown-loader, but don't see a reference outside that package.